### PR TITLE
Invoices - improve search get

### PR DESCRIPTION
**Pull request by GabrielBlondeau-Kirby**
Add documentation on Search and improve the get list.

Moreover, name refactoring on `order_by` to `sort`.

### CODE REVIEW

File: source/includes/_invoices.md [hunk 1]
```diff
@@ -199,7 +199,75 @@ Arguments | Type | Description
 --------- | ---- | ------
 size | `number` | number of items to retrieve
 page | `number` | which page to retrieve. _default page size is 10_
-order_by | `string` | field used for sorting results. If missing default is "-due_date". Could be other parameters in the Invoice model, like "-created" or "status=DONE".
+sort | `string` | field used for sorting results. If missing default is "-due_date". Could be other parameters in the Invoice model, like "-created" or "status=DONE".
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: could you sort for status? Isn't that what the field `status` is for?

File: source/includes/_invoices.md [hunk 2]
```diff
@@ -199,7 +199,75 @@ Arguments | Type | Description
...
+       "amount":123.45,
+       "_id":{
+          "$oid":"57ee9c72d76d431f85111434"
+       },
+       "image_url": "<image-id>/<image-name>.jpg"
+    }
+]
+```
+
+Retrieves a list of Invoices associate with search.
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: Could you please put the description under the header? And one or two more sentences on what it does, and what you could search for.

File: source/includes/_invoices.md [hunk 3]
```diff
@@ -199,7 +199,75 @@ Arguments | Type | Description
...
+
+Arguments | Type | Description
+--------- | ---- | -------
+**query** | `string` | What you want to search, like **amount** or **account_number**
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: Put in all fields that is possible to search for :)

Comment (KristopherJordan) [inline source/includes/_invoices.md]: Put also in id as a possibility

File: source/includes/_issuers.md [hunk 4]
```diff
@@ -340,7 +340,7 @@ Content-type: application/json
 Retrieves a list of Issuers associate with search.
 
 Arguments | Type | Description
---------- | -----------
+--------- | ---- | -------
 **query** | `string` | What you want to search, like **name** or **account_number**
```
Comment (KristopherJordan) [inline source/includes/_issuers.md]: also `id`

File: source/includes/_invoices.md (hunk 1 above)
Comment (GabrielBlondeau-Kirby) [inline source/includes/_invoices.md]: hummm actually we can't search for a specific status but we can sort by `status`

**Pull request merged**